### PR TITLE
salary validation

### PR DESCRIPTION
**Pull request by Dhanasudheer9494**
implemented the  salary valuation method for validating the salary value

### CODE REVIEW

File: employee_project/__pycache__/employee.cpython-37.pyc
Comment (SuryaBhai-Chinna) [inline employee_project/__pycache__/employee.cpython-37.pyc:1]: Remove the file , It's not required

File: employee_project/employee.py [hunk 1]
```diff
@@ -68,3 +73,16 @@ def welcome_page():
...
     return choice
+
+
+def get_salary(msg = "Enter employee salary:"):
+    """ @param: string
+        @desc: Receive the input from the user and validating the data
+        @returns: int
+    """
+    try:
+        inp = int(input(msg))
```
Comment (SuryaBhai-Chinna) [inline employee_project/employee.py]: inp variable name,  change to sal (Every variable name should be a meaningful )

File: employee_project/employee.py [hunk 2]
```diff
@@ -68,3 +73,16 @@ def welcome_page():
...
+    except ValueError:
+        print("invalid input,enter an integer...")
```
Comment (SuryaBhai-Chinna) [inline employee_project/employee.py]: Change the description to "Invalid input, enter a valid input ex: 20000"

File: employee_project/main.py [hunk 3]
```diff
@@ -16,7 +16,7 @@ def main():
             ph_num = input("Enter Employee Phone Number: ")
             address = input("Enter Employee Address : ")
             designation = input("Enter Employee Designation : ")
-            sal = input("Enter Employee Salary: ")
+            sal =get_salary ("Enter Employee Salary: ")
```
Comment (SuryaBhai-Chinna) [inline employee_project/main.py]: give space after assignment operator

File: employee_project/employee.py [hunk 4]
```diff
@@ -68,3 +73,16 @@ def welcome_page():
     print(" Exit the Apllication Enter number           : 4")
     choice = int(input("\nEnter your choice : "))
     return choice
+
+
+def get_salary(msg = "Enter employee salary:"):
```
Comment (SuryaBhai-Chinna) [inline employee_project/employee.py]: Method name get_salary to get_emp_salary_input()

**Review by SuryaBhai-Chinna — COMMENTED** [review]
Fix these comments

File: employee_project/employee.py (hunk 1 above)
Comment (Dhanasudheer9494) [reply employee_project/employee.py]: changed

File: employee_project/employee.py (hunk 2 above)
Comment (Dhanasudheer9494) [reply employee_project/employee.py]: changed description

File: employee_project/main.py (hunk 3 above)
Comment (Dhanasudheer9494) [reply employee_project/main.py]: added

File: employee_project/employee.py (hunk 4 above)
Comment (Dhanasudheer9494) [reply employee_project/employee.py]: changed the method name

File: employee_project/__pycache__/employee.cpython-37.pyc
Comment (Dhanasudheer9494) [reply employee_project/__pycache__/employee.cpython-37.pyc:1]: removed

**Review by SuryaBhai-Chinna — APPROVED** [review]
Changes are looks good for me, Merge the code

**Pull request merged**